qualtran/surface_code/physical_parameters.py

```diff
@@ -0,0 +1,47 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from attrs import field, frozen
+
+
+@frozen
+class PhysicalParameters:
+    """The physical properties of a quantum computer..
+
+    Attributes:
+        t_gate_ns: Clifford gate time (ns).
+        t_meas_ns: Measurement time (bs).
```

Comment:
ns, bs -- should be us ; doesn't match actual class name

Reply:
why `us`? the latencies in both the azure and our models are `~100ns`